### ISSUE TITLE
[Bug Report]: Messages in the console don't appear at bottom

### ISSUE THREAD

**Issue by zLevii** [WitherHosting/Bug-Tracker#5]
### Expected Behaviour

Messages should appear at the bottom of the console

### Final Outcome

Messages appear at the bottom however there's empty space that's just too big

### Steps to Reproduce Issue

```text
Type in the console when ur console box is full and you need to scroll and you will see how it doesn't type at the end

The message below the "see how high it is" just shows you how much space there is between the message and "Type a command"
```

### Screenshots

![image](https://github.com/WitherHosting/Bug-Tracker/assets/44142585/2bdc73b5-e0ea-42c2-9ca5-af296117cb0a)

### Web Origin

Web: WP-01 Version: 3.0.1

**Comment by zLevii** [comment]
This is only an issue on Windows for me. It's perfectly fine on mobile and behaves as intended.

**Comment by RediPanda** [comment]
Looks to be reproduceable when a server is booting up and our uptime timer is loaded into the side panel. I'll check out the issue and hopefully push out a patch in a bit.

**Comment by RediPanda** [comment]
Issue resolved in [a1a840d](https://github.com/WitherHosting/WitherPanel-v3/commit/a1a840d6c90d2f370c19a2feb70b55584c26c6c8), will be fixed on live in the next hotfix!

**Comment by zLevii** [comment]
It seems to be lower however there's still 1 line space as seen here:
![image](https://github.com/WitherHosting/Bug-Tracker/assets/44142585/c3302167-c73f-499a-bd7c-777b8245c33f)
